scripts/run_metrics.sh

```diff
@@ -11,6 +11,27 @@ export BUILD_TARGET=${BUILD_TARGET-${TEST_ARCH}-unknown-linux-gnu}
 WORKLOADS_DIR="$HOME/workloads"
 mkdir -p "$WORKLOADS_DIR"
 
+build_custom_linux() {
+    SRCDIR=$PWD
+    LINUX_CUSTOM_DIR="$WORKLOADS_DIR/linux-custom"
+    LINUX_CUSTOM_BRANCH="ch-5.15.12"
+    LINUX_CUSTOM_URL="https://github.com/cloud-hypervisor/linux.git"
+
+    checkout_repo "$LINUX_CUSTOM_DIR" "$LINUX_CUSTOM_URL" "$LINUX_CUSTOM_BRANCH"
+
+    cp $SRCDIR/resources/linux-config-${TEST_ARCH} $LINUX_CUSTOM_DIR/.config
+
+    pushd $LINUX_CUSTOM_DIR
+    if [ ${TEST_ARCH} == "x86_64" ]; then
+       make bzImage -j `nproc`
```

Comment:
```suggestion
       make -j `nproc`
```

Reply:
I don't think you need to specify `bzImage` in there, especially since we only care about `vmlinux`.

Reply:
Right, this was the original script, but I agree, and also we do not need to specify `make Image` for arm.

Reply:
Done